contracts/test/v0.8/dev/KeeperRegistryDev.test.ts

```diff
@@ -369,6 +369,123 @@ describe('KeeperRegistryDev', () => {
     })
   })
 
+  describe('#pauseUpkeep', () => {
```

Comment:
can we add some tests to `checkUpkeep` and `performUpkeep` to make sure they revert when paused?

Reply:
Done